### PR TITLE
feat(1824): update docs links to new URL structure (remove '/docs')

### LINKED ISSUE #1824: Update docs links to new URL structure (remove '/docs')
## Chore

Docusaurus (the framework for docs.vega.xyz) no longer requires that we have /docs/ in the URL for it to work as a documentation site. This means we can slim down the docs URLs i.e. 

docs.vega.xyz/docs/testnet/api/rest/overview

becomes

docs.vega.xyz/testnet/api/rest/overview

UPDATE: 

This has now been done in the below PR however we just need to adjust this above link as it uses the previous structure:

https://github.com/vegaprotocol/frontend-monorepo/pull/1971/files#diff-c51c4f00f9a83aeb292818bef8e41516981200012464e0c815ddc7a33424cac5R19

### PR DESCRIPTION
**Pull request by sam-keen**
# Related issues 🔗

Closes #1824

### CODE REVIEW

**Review by edd — APPROVED** [review]

File: libs/react-helpers/src/lib/links/links.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const ExternalLinks = {
   GITHUB: 'https://github.com/vegaprotocol/token-frontend',
   DISCORD: 'https://vega.xyz/discord',
   GOVERNANCE_PAGE: 'https://vega.xyz/governance',
-  PROPOSALS_GUIDE: 'https://docs.vega.xyz/docs/mainnet/tutorials/proposals',
+  PROPOSALS_GUIDE: 'https://docs.vega.xyz/mainnet/tutorials/proposals',
```
Comment (dexturr) [inline libs/react-helpers/src/lib/links/links.ts:19]: chore (blocking): this is hardcoded to mainnet, we should move this to the get docs link function aboe

**Review by dexturr — CHANGES_REQUESTED** [review]
Little thing

File: libs/react-helpers/src/lib/links/links.ts (hunk 1 above)
Comment (edd) [reply libs/react-helpers/src/lib/links/links.ts:19]: There's me being shown up as a lazy reviewer

Comment (sam-keen) [reply libs/react-helpers/src/lib/links/links.ts]: Good spot. I've now updated this and the proposal docs links.

**Review by dexturr — APPROVED** [review]

**Pull request merged**